src/dune_rules/lib_info.mli

```diff
@@ -195,11 +197,13 @@ val for_dune_package :
   -> implements:(Loc.t * Lib_name.t) option
   -> default_implementation:(Loc.t * Lib_name.t) option
   -> sub_systems:Sub_system_info.t Sub_system_name.Map.t
-  -> modules:Modules.t
   -> Path.t t
 
 val map_path : Path.t t -> f:(Path.t -> Path.t) -> Path.t t
 
+val map_modules :
+  Path.t t -> f:(Modules.t option -> Modules.t option) -> Path.t t
```

Comment:
Shouldn't this be polymorphic in the path?

Reply:
addressed in [8375c33](https://github.com/ocaml/dune/pull/6605/commits/8375c33f2ccc9ea1680a1b76e3c4323bbb6a0fe3)